docs/sources/tempo/traceql/_index.md

```diff
@@ -94,6 +94,8 @@ The following table shows the current available scoped intrinsic fields:
 | `event:timeSinceStart`  | duration    | time of event in relation to the span start time                | `{ event:timeSinceStart > 2ms}`        |
 | `link:spanID`           | string      | link span id using hex string                                   | `{ link:spanID = "0000000000000001" }` |
 | `link:traceID`          | string      | link trace id using hex string                                  | `{ link:traceID = "1234567890abcde" }` |
+| `scope:name`            | string      | instrumentation scope name                                      | `{ scope:name = "grpc" }`              |
```

Comment:
I thought the thing at the left of `:` was a scope , i.e. `link` is a possible value for scope. 
So is very confusing for me that this is called `scope` , shouldn't it be called `instrumentation` ?
`instrumentation:name` `instrumentation:version` more clearly define what this points at IMHO

Reply:
this is a good call out. we have discussed this with our internal team and they believe that otel does not see this as an instrumentation only field. it just happens to be used that way and they intend for it to be used as a more generic "scope". 

agree that scope scope is strange. let me summon @jpkrohling. he may have more insight.

Reply:
An instrumentation scope can have a name and a version, in addition to arbitrary key/value pairs, so I think this is valid as is: `{ scope:name = "grpc" }`.

Reference: https://github.com/open-telemetry/opentelemetry-proto/blob/40b3c1b746767cbc13c2e39da3eaf1a23e54ffdd/opentelemetry/proto/common/v1/common.proto#L69-L81

Reply:
Are end-users exposed to "scope scope"? As an OTel user, I immediately thought of [InstrumentationScope](https://github.com/open-telemetry/opentelemetry-proto/blob/v1.3.2/opentelemetry/proto/common/v1/common.proto#L69-L81) when I saw "scope:name" here. Unless there's an explicit source of confusion, I'd keep "scope:name".

Reply:
> Are end-users exposed to "scope scope"?

Not directly although there may be some awkward wording in the docs.

Reply:
Personally `scope:name` is a generic term that says nothing to me. 
I can imagine for people better versed in opentelemetry parlance it is very obvious. 
I am very sure that around here I will need to tell people that "scope:name means the name of the instrumentation/area that created the span". or such explanation as I do not think it is clear to more casual users of observability tools.

Reply:
Would it help to have these differences called out in the doc to avoid confusion? Or would that just create more confusion?

Reply:
Docs will definitely need to be there.
When someone reads a query or an autocomplete with `span:duration` , that person needs to know almost nothing about tracing to understand what that would mean, without resorting to reading documentation.

I think when someone reads `scope:name` that person will definitely need to go to documentation to know what would that mean.

Reply:
@jcarres-mdsol thank you for pointing this out. I've changed the syntax to `instrumentation:name` and `instrumentation:version` 🚀